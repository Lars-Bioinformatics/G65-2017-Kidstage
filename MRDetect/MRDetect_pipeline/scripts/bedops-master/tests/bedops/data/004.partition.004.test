chr1	279	280
chr1	280	281
chr1	281	282
chr1	310	311
chr1	310	320
chr1	311	312
chr1	312	313
chr1	312	318
chr1	313	314
