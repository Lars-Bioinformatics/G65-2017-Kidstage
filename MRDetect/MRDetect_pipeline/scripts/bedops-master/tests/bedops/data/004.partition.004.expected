chr1	279	280
chr1	280	281
chr1	281	282
chr1	310	311
chr1	311	312
chr1	312	313
chr1	313	314
chr1	314	318
chr1	318	320
